app/controllers/lists_controller.rb

```diff
@@ -0,0 +1,50 @@
+class ListsController < ApplicationController
+    before_action :find_board_list, only: [:edit, :update, :destroy] 
+
+    def new
+        @board = Board.find(params[:board_id])
+        @list = @board.lists.new
+    end
+
+    def create
+        @board = Board.find(params[:board_id])
+        @list = @board.lists.create(create_params)
+        if @list.save
+            redirect_to board_path(@board), notice: 'List created'
+        else
+            render :new
+        end
+    end
+
+    def edit
+    end
+
+    def update
+          if @list.update(name: params[:list][:name])
```

Comment:
ustaw sobie wciecia na 2 spacje

Reply:
trzymaj sie strong parameters typu update_params nawet jak masz tylko jedna wartosc. Pozniej jak ktos przyjdzie i bedzie dodawal atrybuty to w jednym miejscu doda

Reply:
def update_params
        params.update(name: params[:list][:name])
    end

 def update
          if @list.update_attributes
            redirect_to boards_path(@board), notice: 'List name changed!'
          else
            render :edit
          end
    end


??